CustomAnalysisRules.ruleset

```diff
@@ -5,7 +5,7 @@
   <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp.Features" RuleNamespace="Microsoft.CodeAnalysis.CSharp.Features">
     <Rule Id="IDE0003" Action="Warning" />
     <Rule Id="IDE0004" Action="Warning" />
-    <Rule Id="IDE0005" Action="Warning" />
```

Comment:
>Warning [](start = 31, length = 7)

I actually like this rule, it keep your using statement clean. Any reason to ignore it?

Reply:
I the rule too, but I don't like the way it shows up in your IDE sporadically based on whether or not you have the file open. I would prefer it to run on build, for for that it appears we need to turn XML documentation generation. Maybe there is another rule that covers this?

Reply:
And if we turn on XML documentation generation, we get a bunch of errors because of malformed xml, missing documentation, etc. I am also concerned that this will hurt compilation speed.
If you want to keep this, I can just put back in IDE0005  (at least for now).

Here is the compiler bug: https://github.com/dotnet/roslyn/issues/41640

Reply:
I've reverted the change.